lightningnetwork/lnd
# lnd: remove deprecated endpoints `sendpayment`, `sendtoroute`, `sendtoroutesync`, and `sendpaymentsync`

## Change Description
This pull request removes the deprecated endpoints `sendpayment`, `sendtoroute`, `sendtoroutesync`, and `sendpaymentsync` in `lightning.proto`, `rpcserver.go`, `lntest/rpc/lnd.go`, and its related test cases in `itest/lnd_routing_test.go` and `itest/lnd_channel_policy_test.go`.

Closes #8175

## Steps to Test
I ran the test cases using `make check` to make sure there are no regressions and all the test cases passed.

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
